sale_order_product_availability_inline/views/sale_views.xml

```diff
@@ -15,6 +15,16 @@
                     "warehouse": parent.warehouse_id}
                 </attribute>
             </xpath>
+
+            <xpath
+                expr="//field[@name='order_line']/list//field[@name='product_template_id']"
+                position="attributes"
+            >
+                <attribute name="context" operation="update">
+                    {"so_product_stock_inline": True,
+                    "warehouse": parent.warehouse_id}
```

Comment:
Isn't there here other way for not overriding the context and being fragile if Odoo changes this context upstream (or in new versions)?